src/components/MediaCard.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { Card, CardActionArea, CardActions, CardContent, CardMedia, Button, Typography, makeStyles, Snackbar } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import { Alert } from '@material-ui/lab';
+
+const useStyles = makeStyles({
+    root: {
+        maxWidth: 345,
+    },
+    media: {
+        height: 140,
+    },
+})
+
+export default function MediaCard(props) {
+    const [showDescription, setShowDescription] = useState(props.showDescription)
+    const [saved, setSaved] = useState(props.saved)
+    const [id, setId] = useState(props.id)
+    const [addedAlert, setAddedAlert] = useState(false)
+    const [removedAlert, setRemovedAlert] = useState(false)
+
+    const { title } = props
+    const { href } = props
+    const { description } = props
+    const classes = useStyles()
+
+    const savePicToDB = async () => {
+        const newId = await props.savePicToDB({ title: title, href: href, description: description })
```

Comment:
in an object, when the keys' names = values' name, you can do: `{title, href, description}` instead of - `{ title: title, href: href, description: description }`